cmd/podman/restart.go

```diff
@@ -57,6 +57,18 @@ func restartCmd(c *cliconfig.RestartValues) error {
 	}
 	defer runtime.DeferredShutdown(false)
 
+	if c.AutoUpdate {
```

Comment:
Maybe, adding a new command is the way to go. It would allow for adding flags in the future (e.g., `--rollback`) and could be beneficial for docs where can give more verbose examples (without exploding the restart man page).